db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_release.sql

```diff
@@ -20,3 +20,20 @@ ALTER TABLE pms.partner_h ADD COLUMN addl_info character varying;
 
 INSERT INTO pms.partner_type (code,partner_description,is_policy_required,is_active,cr_by,cr_dtimes,upd_by,upd_dtimes,is_deleted,del_dtimes) VALUES
 	 ('Print_Partner','Print Partner',true,true,'superadmin',now(),NULL,NULL,false,NULL);
+
+UPDATE pms.partner_type SET is_active=true, upd_dtimes =now(), upd_by ='superadmin' WHERE code='MISP_Partner';
```

Comment:
we want to activate all?

Reply:
it is not is_active, it should be is_policy_required set to true.